packages/replay-next/src/utils/protocol.ts

```diff
@@ -437,3 +430,7 @@ export function findProtocolObjectPropertyValue<Type>(
 ): Type | null {
   return findProtocolObjectProperty(sourceObject, name)?.value ?? null;
 }
+
+export function isArrayElement(property: ProtocolProperty): boolean {
```

Comment:
This name confuses me. I know this is a nit, but can you elaborate on why you chose "array element"? I would assume that meant something more like "The value contained is an array"

Reply:
I couldn't think of a better word for an item in an array, I usually call them array elements.